src/content/tutorials/observability/data-ingestion/metrics/_index.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Ingest metrics into the Observability Platform
+linkTitle: Metrics
+description: How to ingest new metrics in the Observability Platform
+menu:
+  principal:
+    parent: tutorials-observability-data-ingestion
+    identifier: tutorials-observability-data-ingestion-metrics
+weight: 50
+last_review_date: 2024-07-11
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-atlas
+user_questions:
+  - What is a ServiceMonitor?
+  - What is a PodMonitor?
+  - How do I ingest new metrics?
+  - Why do my clusters run the Prometheus Operator?
+---
+
+By default, all Giant Swarm clusters are equiped with the [Prometheus Operator](https://prometheus-operator.dev/) and a set monitoring agents (multiple [Prometheus](https://prometheus.io/) shards in `agent mode`) to be able to collect and forward cluster critical and workload metrics to a [Grafana Mimir](https://grafana.com/oss/mimir/) instance running on the management cluster.
+
+As no workload is the same, especially in the way it exposes its metrics, we decided to base our monitoring configuration on the `ServiceMonitor` and `PodMonitor` CRDs provided by the Prometheus Operator. Those allow us to:
+
```

Comment:
remove this line break so the 

> Those allow you to: 

and the bullet points are one after another

Reply:
This line break is required by the linter :D